### ISSUE TITLE
обработать еще одну модельную галактику

### ISSUE THREAD

**Issue by Amarchuk** [alekskryzh/X-galaxies#14]
Надо прочертить максимумы, вычитать ничего не нужно. Ссылка на файл: https://disk.yandex.ru/d/LDg3z44P45pi3w

Разницу с предудыщими объясним на созвоне.

**Comment by alekskryzh** [comment]
Сделано для трёх эллиптичностей: 0.3,0.4,0.5 (им соответствуют суффиксы _0_3 и т.д.). Вообще, как мне кажется, для вычетов основной скрипт (автоматика) работает вполне неплохо, в ручном режиме (протыкивание максимум) есть трудность нахождения максимумов из-за интенсивности в плоскости диска, особенно, для меньшей эллиптичностей. Для большей эллиптичности в ручном режиме х-структура получается слегка изогнутой, в то время как автоматика даёт структуру прямее (за исключением концов структуры). 

Ноутбук: https://github.com/alekskryzh/X-galaxies/blob/main/decomp_results/plot_x_peaks_Mh_0316.ipynb

Архив: https://disk.yandex.ru/d/PMKCBNFB7gE68Q

**Comment by Amarchuk** [comment]
> Для большей эллиптичности в ручном режиме х-структура получается слегка изогнутой, в то время как автоматика даёт структуру прямее (за исключением концов структуры).

Да, это хорошо видно по углам. Вот только автоматика справляется гораздо хуже в среднем на реальных галактиках. 

Результаты интересные.

**Comment by greenyellowcat** [comment]
Возьмите, пожалуйста, побольше эллиптичность все таки (0.7-0.8). При ручном протыкивании профиль сильно плоский у пика, и плохо определяется максимум.

**Comment by alekskryzh** [comment]
Добавлены результаты с эллиптичностями 0.7,0.8. 

С эллиптичностями 0.7,0.8 автоматика справилась хуже (с 0.8 и вовсе не справилась). Для 0.7 подозреваю, что плохие точки связаны с попыткой учесть дисковую составляющую (уши) там, где ничего нет (нулевые значения), это для внешних эллипсов, большая полуось которых как раз выходит за пределы ушей. Хотя это предположение не работает для 0.8. 

Архив: https://disk.yandex.ru/d/iwqBln0UYb5LKQ

Ноутбук: https://github.com/alekskryzh/X-galaxies/blob/881022e9c5ea2a65b74e094b623d69b7ffdf0377/decomp_results/plot_x_peaks_Mh_0317.ipynb
